src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/model/managedindexmetadata/ActionProperties.kt

```diff
@@ -28,17 +28,23 @@ import org.elasticsearch.common.xcontent.XContentParserUtils.ensureExpectedToken
 /** Properties that will persist across steps of a single Action. Will be stored in the [ActionMetaData]. */
 data class ActionProperties(
     val maxNumSegments: Int? = null,
-    val snapshotName: String? = null
+    val snapshotName: String? = null,
+    val rollupId: String? = null,
+    val hasRollupFailed: Boolean? = null
```

Comment:
Possible to namespace these under "rollup" since they are related? Admittedly this was just a dumping ground of random values that needed to persist across step executions, but worried it'll continue to just grow and get confusing to reason about, especially if some properties are tied together.

Reply:
will add a TODO for this, agree this can blow up if we add more fields to this